### PR TITLE
Use multiple if-let unwrapping

### PR DESCRIPTION
**Pull request by ikesyo**

### CODE REVIEW

File: ReactiveCocoa/Swift/FoundationExtensions.swift [hunk 1]
```diff
@@ -32,12 +32,12 @@ extension NSURLSession {
 	public func rac_dataWithRequest(request: NSURLRequest) -> SignalProducer<(NSData, NSURLResponse), NSError> {
 		return SignalProducer { observer, disposable in
 			let task = self.dataTaskWithRequest(request) { (data, response, error) in
-				if data == nil || response == nil {
-					sendError(observer, error)
-				} else {
-					let value = (data!, response!)
+				if let data = data, response = response {
+					let value = (data, response)
 					sendNext(observer, value)
```
Comment (neilpa) [inline ReactiveCocoa/Swift/FoundationExtensions.swift]: `sendNext(observer, (data, response))`

**Comment by neilpa** [comment]
Thanks. One minor comment.

**Comment by ikesyo** [comment]
Thanks, just fixed it :zap:

**Comment by neilpa** [comment]
:tada:

**Pull request merged**